src/services/ImageRepository.cpp

```diff
@@ -10,8 +10,8 @@
 namespace ra {
 namespace services {
 
-constexpr char* DefaultBadge = "00000";
-constexpr char* DefaultUserPic = "_User";
+inline constexpr const char* DefaultBadge = "00000";
+inline constexpr const char* DefaultUserPic = "_User";
```

Comment:
With conformance on, what was here was wrong.
Straight from Visual Studio: `info: Conversion from string literal loses const qualifier.`
If `constexpr `existed in C it would be correct, but there's a difference between `char*` (pointer to an array of characters) and` const char* ` (pointer to a null-terminated string) in C++. if you aren't sure what the value_type is just put auto.